.env.example

```diff
@@ -1,5 +1,26 @@
 #For more information on .env files, their content and format: https://pypi.org/project/python-dotenv/
 
+######################################
+########### Admin Settings ###########
+######################################
+
+# Enable Admin GUI, available for the first user in LNBITS_ADMIN_USERS if available
+# Warning: Enabling this will make LNbits ignore this configuration file. Your settings will
+# be stored in your database and you will be able to change them only through the Admin UI.
+# Disable this to make LNbits use this config file again.
```

Comment:
i think this is not true, disabling this will do nothing, as soon as its creates the settings table it will use it, so you will need to truncate settings and disabling this.

but honestly i don't have a clue why you should ever switch back or between them.